platform/security/core/security-core-impl/src/test/groovy/ddf/security/samlp/impl/LogoutMessageSpecTest.groovy

```diff
@@ -24,12 +23,10 @@ class LogoutMessageSpecTest extends Specification {
     LogoutMessageImpl logoutMessage;
 
     final String ENCRYPT_PREFIX = "ENCRYPTED "
-
     final String NAME_ID = "MyNameId"
-
     final String ISSUER_ID = "MyIssuerId"
-
     final String IN_RESPONSE_TO = "InResponseToID"
+    final List SESSION_INDEXES = Collections.emptyList();
```

Comment:
✏️  Nit: Blank line before to be consistent?

Reply:
They are all on the same line. I think github has to format it that way to show that lines were removed from the previous version. This is what it actually looks like;

```  final String ENCRYPT_PREFIX = "ENCRYPTED "
    final String NAME_ID = "MyNameId"
    final String ISSUER_ID = "MyIssuerId"
    final String IN_RESPONSE_TO = "InResponseToID"
    final List SESSION_INDEXES = Collections.emptyList();
```